tests/testthat/test-pstr_add_reductions.R

```diff
@@ -1,8 +1,71 @@
-test_that("hasn't changed", {
-  with_reductions <- pstr_companies |>
-    pstr_add_reductions(pstr_ep_weo, pstr_weo_2022) |>
-    # FIXME: Address warning
-    suppressWarnings()
-  out <- format_robust_snapshot(with_reductions)
-  expect_snapshot(out)
```

Comment:
As you wrote more robust and meaningful tests, it's now time to remove the brittle and non-specific snapnshot test. It's then a good idea to run a report of test coverage to confirm all lines are still 100% covered by tests -- which allow us to refactor aggresively knowing that if we break something important then our tests will complain.

![image](https://user-images.githubusercontent.com/5856545/229842872-4c94eaaa-da4b-469a-b693-1a170f9c502d.png)

Reply:
:O wow !!